data/sql/updates/pending_db_world/rev_1623792334812044570.sql

```diff
@@ -0,0 +1,13 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1623792334812044570');
+
+SET @POOL = 60004;
```

Comment:
I'm gonna start using these set variables for my queries aswell!

Reply:
@claudiodfc

I took inspiration from PRs like #5071.

I like the idea of not repeating values if they are used multiple times; makes it easier to read and avoids typos. :D

Reply:
Also the readability is much better!